.github/workflows/check_python_methods.py

```diff
@@ -50,19 +53,61 @@ def parse(type, names):
             service = "cloud"
 
         # Find all python methods objects on Python docs site soup
-        py_methods_sdk_docs = soup.find_all("dt", class_="sig sig-object py")
-        py_methods_sdk_docs_filtered = []
+        py_methods_sdk_docs = soup.find_all("dl", class_="py method")
+        py_methods_sdk_docs_filtered_ids = []
 
         # Get ids and filter list
         for tag in py_methods_sdk_docs:
-            id = tag.get('id')
+            tag_sigobject = tag.find("dt", class_="sig sig-object py")
+            id = tag_sigobject.get('id')
 
             if not id.endswith("Client") and not id.endswith(".client") \
             and not id.endswith("SUBTYPE") and not id.endswith(".from_robot") \
             and not id.endswith(".get_resource_name") and not id.endswith(".get_operation") \
             and not id.endswith(".LOGGER") and not id.endswith("__"):
-                py_methods_sdk_docs_filtered.append(id)
-    
+                py_methods_sdk_docs_filtered_ids.append(id)
+
+            # Get methods information
+            method_text = []
+            
+            # Descriptions
+            # UNCOMMENT IF YOU WANT TO CATCH ALL EXTRA RETURNS INFORMATION/RETURNS DESCRIPTION
+            # description_elements = tag.find_all("p")
+            # if description_elements:
+            #     index = 0
+            #     for element in description_elements:
+            #         if index >= 1:
+            #             method_text.append("-" + element.text + "\n")
+            #         else:
+            #             method_text.append(element.text + "\n")
+            #         index += 1
+            description_element = tag.find("p")
+            if description_element:
+                method_text.append(description_element.text + "\n\n")
+            
+            method_text.append("**Parameters:** \n")
+
+            # Parameters
+            param_element = tag_sigobject.find_all("em", class_="sig-param")
+            if param_element:
+                for element in param_element:
+                    method_text.append("- " + element.text + "\n")
+
+            method_text.append("\n **Returns:** \n")
+
+            # Returns
+            return_element = tag_sigobject.find_all("span", class_="sig-return")
+            if return_element:
+                for element in return_element:
+                    method_text.append("- " + element.text + "\n")
+
+            method_text.append(f"\n For more information, see the [Python SDK Docs]({url})")
```

Comment:
You can make the full url with the id. You could also obtain the rest of the link from the headerlink class:
```suggestion
            method_text.append(f"\n For more information, see the [Python SDK Docs]({url}#{id})")
```